test/OpenSpaceToolkit/Astrodynamics/Trajectory/State.test.cpp

```diff
@@ -14,17 +16,33 @@ using ostk::physics::time::Instant;
 using ostk::physics::time::Scale;
 using ostk::physics::units::Length;
 
+using ostk::astro::trajectory::CoordinatesBroker;
+using ostk::astro::trajectory::CoordinatesSubset;
 using ostk::astro::trajectory::State;
 
 TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_State, Constructor)
 {
     {
         const Instant instant = Instant::DateTime(DateTime(2018, 1, 1, 0, 0, 0), Scale::UTC);
-        VectorXd v(6);
-        v << 1.0, 2.0, 3.0, 4.0, 5.0, 6.0;
-        EXPECT_NO_THROW(State state(instant, v, Frame::GCRF()));
+        VectorXd coordinates(6);
+        coordinates << 1.0, 2.0, 3.0, 4.0, 5.0, 6.0;
+        const Shared<const CoordinatesBroker> broker = std::make_shared<const CoordinatesBroker>(
+            CoordinatesBroker({CoordinatesSubset::Position(), CoordinatesSubset::Velocity()})
+        );
+
+        EXPECT_NO_THROW(State state(instant, coordinates, Frame::GCRF(), broker));
     }
 
+    // {
+    //     const Instant instant = Instant::DateTime(DateTime(2018, 1, 1, 0, 0, 0), Scale::UTC);
+    //     std::vector<double> coordinates = {1.0, 2.0, 3.0, 4.0, 5.0, 6.0};
+    //     const Shared<const CoordinatesBroker> broker = std::make_shared<const CoordinatesBroker>(
+    //         CoordinatesBroker({CoordinatesSubset::Position(), CoordinatesSubset::Velocity()})
+    //     );
+
+    //     EXPECT_NO_THROW(State::fromStdVector(instant, coordinates, Frame::GCRF(), broker));
```

Comment:
I added 5 tests to check the static constructor but I had to comment out as the compiler didnt like them. 

I wasn't able to make anything out of the error message. So they are commented out for now. Any idea what I might be doing wrong?